R/joins.R

```diff
@@ -32,34 +32,42 @@
 NULL
 
 #' @rdname join.tbl_dt
-inner_join.data.table <- function(x, y, by = NULL, copy = FALSE, ...){
+inner_join.data.table <- function(x, y, by = NULL, copy = FALSE, 
+                                  suffix = c(".x", ".y"), ...){
   by <- dplyr::common_by(by, x, y)
   y <- dplyr::auto_copy(x, y, copy = copy)
-  out <- merge(x, y, by.x = by$x, by.y = by$y, all = FALSE, allow.cartesian = TRUE)
+  out <- merge(x, y, by.x = by$x, by.y = by$y, all = FALSE, suffixes = suffix, 
```

Comment:
I think if the first argument is on a newline, all arguments should be on their own line and the closing parentheses. Otherwise you fill the lines to the 72nd or 80th column and leave the closing parenthese with the final argument.